src/timeMachine/components/Vis.tsx

```diff
@@ -82,42 +75,6 @@ const TimeMachineVis: FC<Props> = ({
     'time-machine--view__empty': noQueries,
   })
 
-  // Handles deadman check edge case to allow non-numeric values
-  const viewRawData =
-    isViewingRawData ||
-    (type === 'check' &&
-      giraffeResult.table.getColumnType('_value') !== 'number' &&
-      !!giraffeResult.table.length)
-
-  if (viewRawData && files && files.length && !isFlagEnabled('simpleTable')) {
-    const [parsedResults] = files.flatMap(fromFlux)
-    return (
-      <div className={timeMachineViewClassName}>
-        <ErrorBoundary>
-          <EmptyQueryView
-            loading={loading}
-            errorMessage={errorMessage}
-            errorFormat={ErrorFormat.Scroll}
-            hasResults={!!parsedResults?.table?.length}
-            isInitialFetch={isInitialFetch}
-          >
-            <AutoSizer>
-              {({width, height}) => {
-                return (
-                  <RawFluxDataTable
-                    parsedResults={parsedResults}
-                    width={width}
-                    height={height}
-                  />
-                )
-              }}
-            </AutoSizer>
-          </EmptyQueryView>
-        </ErrorBoundary>
```

Comment:
Wait, you're gonna need this back in here, except replace the RawFluxDataTable with SimpleTable I think